src/Illuminate/Database/Concerns/ManagesTransactions.php

```diff
@@ -81,6 +93,35 @@ protected function handleTransactionException($e, $currentAttempt, $maxAttempts)
         throw $e;
     }
 
+    /**
+     * Handle an exception encountered when committing a transaction.
+     *
+     * @param  \Exception  $e
+     * @param  int  $currentAttempt
+     * @param  int  $maxAttempts
+     * @return void
+     *
+     * @throws \Exception
```

Comment:
This class doesn't throws an exception. Remove this annotation please.

Reply:
@driesvints It rethrows `$e` at line 123.

Reply:
@jansennerd10 we don't document rethrown exceptions.

Reply:
@driesvints ah, okay. Fixed in 53ae6e9.